pkg/helmer/helm_cli.go

```diff
@@ -493,7 +497,7 @@ func (h *HelmCLI) ListReleases(ns string) (map[string]ReleaseSummary, []string,
 			}
 		}
 	}
-	slice.SortStrings(keys)
+	sort.Strings(keys)
```

Comment:
They are the same: https://github.com/kubernetes/kubernetes/blob/978d9683f5c253cf62225dc0656c0bfeb2a7d339/pkg/util/slice/slice.go#L37, it's also not recommended to import kubernetes package.